doc/auth/saml.md

```diff
@@ -0,0 +1,133 @@
+## Overview
+
+This guide will walk through using Pachyderm's SAML support.
+These features aren't integrated into mainline Pachyderm yet and aren't
```

Comment:
Hitting the merge button would make this untrue, right?

Reply:
Removed :-)